lib/CLI.js

```diff
@@ -156,13 +156,13 @@ Example:
 
     // Show help
     } else if (this.argv.help === true) {
-      logger.silly('Printing help.');
+      logger.debug('Printing help.');
```

Comment:
Are these messages really warranted? (same with pritning version, etc).

It should be pretty clear already